modules/mbedtls/Kconfig

```
@@ -202,6 +202,13 @@ config MBEDTLS_HEAP_SIZE
 	  be needed. For some dedicated and specific usage of mbedtls API, the
 	  1000 bytes might be ok.
 
+config MBEDTLS_INIT
+	bool "Initialize mbed TLS automatically at boot"
```

No reason for `automatically` here.

If something is initialized at boot, then it must be kind of automatically.
```suggestion
	bool "Initialize mbed TLS at boot"
```

---

agreed, i'll sort it